compiler/rustc_const_eval/src/const_eval/error.rs

```diff
@@ -119,7 +119,10 @@ pub fn get_span_and_frames<'tcx>(
     if frames.len() > 0 {
         frames.remove(0);
     }
-    if let Some(last) = frames.last_mut() {
+    if let Some(last) = frames.last_mut()
+        // If the span is not going to be printed, we don't want the span label for `is_last`.
```

Comment:
Please rename the `is_last` field to `span_label` or so, that seems to better reflect its intent, given this comment.